model/chunks/chunkLocator.go

```diff
@@ -0,0 +1,23 @@
+package chunks
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// Locator is used to locate a chunk by providing the execution result the chunk belongs to as well as the chunk index within that execution result.
```

Comment:
```suggestion
// Locator is used to locate a chunk by providing the execution result the chunk belongs to as well as the chunk index within that execution result.
// Since a chunk is unique by the result ID and its index in the result's chunk list.
```